.github/workflows/ci.yml

```diff
@@ -13,27 +13,46 @@ jobs:
         name: [ "Linux x64 (Ubuntu 20.04) - GCC 10" ]
         os: [ ubuntu-20.04 ]
         sanitize: [ false ]
-        build:
-          - { cc: gcc, cxx: g++, linker: ld, shell: bash }
+        build: { cc: gcc, cxx: g++, linker: ld }
+        shell: bash
+        prefix: "/usr/local"
+        sudo: "sudo"
+
         include:
           - name: "Linux x64 (Ubuntu 20.04) - Clang 10 with ASan and UBSan"
             os: ubuntu-20.04
             sanitize: true
-            build: { cc: clang-10, cxx: clang++-10, linker: ld.lld-10, shell: bash }
+            build: { cc: clang-10, cxx: clang++-10, linker: ld.lld-10 }
+            shell: bash
+            prefix: "/usr/local"
+            sudo: "sudo"
+
           - name: "macOS (10.15) - Xcode 12.3"
             os: macos-10.15
             sanitize: false
-            build: { cc: clang, cxx: clang++, linker: ld.lld, shell: bash }
+            build: { cc: clang, cxx: clang++, linker: ld.lld }
+            shell: bash
+            prefix: "/usr/local"
+            sudo: "sudo"
+
           - name: "macOS (10.15) - Xcode 12.3 - M1"
             os: macos-m1
             sanitize: false
-            build: { cc: clang, cxx: clang++, linker: ld.lld, shell: "/usr/bin/arch -arch arm64e /bin/bash -l {0}" }
+            build: { cc: clang, cxx: clang++, linker: ld.lld }
+            shell: "/usr/bin/arch -arch arm64e /bin/bash -l {0}"
+            prefix: "/Users/Administrator/vips"
+            sudo: ""
 
     defaults:
       run:
         shell: ${{ matrix.build.shell }} 
```

Comment:
```suggestion
        shell: ${{ matrix.shell }} 
```

Reply:
You're right, I forgot to move that out of build.